compile_all_deps.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# --- begin runfiles.bash initialization v2 ---
+# Copy-pasted from the Bazel Bash runfiles library v2.
+set -uo pipefail; f=bazel_tools/tools/bash/runfiles/runfiles.bash
+source "${RUNFILES_DIR:-/dev/null}/$f" 2>/dev/null || \
+  source "$(grep -sm1 "^$f " "${RUNFILES_MANIFEST_FILE:-/dev/null}" | cut -f2- -d' ')" 2>/dev/null || \
+  source "$0.runfiles/$f" 2>/dev/null || \
+  source "$(grep -sm1 "^$f " "$0.runfiles_manifest" | cut -f2- -d' ')" 2>/dev/null || \
+  source "$(grep -sm1 "^$f " "$0.exe.runfiles_manifest" | cut -f2- -d' ')" 2>/dev/null || \
+  { echo>&2 "ERROR: cannot find $f"; exit 1; }; f=; set -e
+# --- end runfiles.bash initialization v2 ---
+
+set -eu
+
+mac="$(rlocation rp_grpcio/requirements.update)"
+linux="$(rlocation rp_grpcio/compile_linux_deps.executable)"
+
+$mac ./requirements.in requirements_darwin.txt None None None "//:requirements.update" --allow-unsafe
+
+$linux --norun 
```

Comment:
Separate importing from running because the executable target echos some messages to stdout so we couldn't redirect it back to requirements_linux.txt